skyportal/facility_apis/atlas.py

```diff
@@ -196,7 +202,11 @@ def commit_photometry(
         from skyportal.handlers.api.photometry import add_external_photometry
 
         if len(df.index) > 0:
-            add_external_photometry(data_out, request.requester)
+            ids, _ = add_external_photometry(
+                data_out, request.requester, duplicates=duplicates
+            )
+            if ids is None:
+                raise ValueError('Failed to commit photometry')
```

Comment:
Similarly, should this be a status?

Reply:
I think it's good to raise that error, and let the method upstream of that decide what status should be set when failing to add the photometry. For some application you might want to just set it as failed and move on, for others you might want to retry again.

Reply:
The queue will use that to set the status.

Reply:
ok!